synfig-core/src/tool/joblistprocessor.cpp

```diff
@@ -264,23 +264,41 @@ void process_job (Job& job)
 	else
 	{
 		VERBOSE_OUT(1) << _("Rendering...") << std::endl;
-		std::chrono::system_clock::time_point start_timepoint =
-            std::chrono::system_clock::now();
 
-		// Call the render member of the target
-		if(!job.target->render(&p))
-			throw (SynfigToolException(SYNFIGTOOL_RENDERFAILURE, _("Render Failure.")));
+		bool should_print_benchmarks = SynfigToolGeneralOptions::instance()->should_print_benchmarks();
+		double dur = 0.f;
+		int repeats = SynfigToolGeneralOptions::instance()->get_repeats();
+
+		for(int i = 0; i < repeats; i++)
+		{
+			std::chrono::system_clock::time_point start_timepoint =
```

Comment:
I suggest to use `steady_clock` instead of `system_clock`, because it's monotonic.